### PR TITLE
Add VDX support to dicom2trip

### LINKED ISSUE #222: dicom2trip does not convert structures
attempted with a pinnacle created dicom file found here:
https://subversion.assembla.com/svn/slicerrt/trunk/SlicerRt/data/pinnacle3-9.9-phantom/
``` dicom2trip dicomdir/  foobar000```

produces only the .ctx files

### PR DESCRIPTION
**Pull request by nbassler**
Closes #222

### CODE REVIEW

**Comment by nbassler** [comment]
How to get rid of all the changes from versioneer which were accidentally commited?

**Comment by nbassler** [comment]
@grzanka MR #224 should me merged first. Then this one will work.

**Comment by grzanka** [comment]
Versioneer stuff removed

File: pytrip/utils/dicom2trip.py [hunk 1]
```diff
@@ -45,8 +45,17 @@ def main(args=sys.argv[1:]):
     c = pt.CtxCube()
     c.read_dicom(dcm)
 
+    v = pt.VdxCube(None,cube=c)
+
+    try:
+        v.read_dicom(dcm)
+        v.write_trip_data(basename + ".vdx")
+    except:
+        pass
```
Comment (grzanka) [inline pytrip/utils/dicom2trip.py]: This shouldn't pass silently. At least some debug error messege should be added

**Comment by nbassler** [comment]
@grzanka 
we could actually fix #190 now as well: omit the unused "contents" argument.

**Pull request merged**